backend/src/api/model/search/mod.rs

```diff
@@ -71,6 +71,12 @@ impl SearchResults<search::Series> {
     }
 }
 
+#[derive(juniper::GraphQLInputObject)]
+pub(crate) struct Filters {
+    item_type: Option<String>,
```

Comment:
This shouldn't be a `String`, but an enum explicitly listing the options. You can check other enums in `schema.graphql` for that. And given that it's already an `Option`, the `all` option probably doesn't need to be part of the enum.

Reply:
Alrighty, so I did that now but kinda struggled to make it work in the frontend. I got there in the end but I can imagine that you might find a better solution.